packages/lab/src/window/ElectronWindow.tsx

```diff
@@ -7,15 +7,24 @@ import { forwardRef, useCallback, useEffect, useRef, useState } from "react";
 import ReactDOM from "react-dom";
 import { isDesktop, Window as SaltWindow, WindowProps } from "../window";
 
-import "./ElectronWindow.css";
 import { useWindowParentContext, WindowParentContext } from "./desktop-utils";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import electronWindowCss from "./ElectronWindow.css";
 
 const Window = forwardRef<HTMLDivElement, WindowProps>(function ElectronWindow(
   { className, children, id = "dialog", open = true, style = {}, ...rest },
   forwardedRef
 ): JSX.Element | null {
   const { top, left, position, ...styleRest } = style;
 
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-electron-window",
```

Comment:
I think even though we will remove this Window, we can't until the positioning stuff is done